templates/banners/banner.html

```diff
@@ -2,16 +2,16 @@
 {% if banners %}
     <div class="banners-container">
         {% for banner in banners %}
-            <div class="alert alert-{{ banner.type }} alert-vertical sm:alert-horizontal" role="alert" data-banner-id="{{ banner.id }}">
-                {% if banner.title %}
-                    <strong>{{ banner.title }}</strong>:
-                {% endif %}
-                {{ banner.message|render_markdown }}
-                <div>
-                    <button class="btn btn-sm btn-ghost dismiss-banner"
-                            data-banner-id="{{ banner.id }}"
-                            aria-label="Close">x</button>
+            <div class="alert alert-{{ banner.type }} alert-vertical sm:alert-horizontal w-full flex items-center justify-between" role="alert" data-banner-id="{{ banner.id }}">
```

Comment:
It's not a recommended practice to dynamically construct classes in the templates because it prevents tailwind from finding them: https://tailwindcss.com/docs/detecting-classes-in-source-files#dynamic-class-names

In this case it's OK because we have these alert classes always included: https://github.com/dimagi/open-chat-studio/blob/d92fb4e3f418e9bff3d0d5f0705033b4b3af21a4/assets/styles/site-tailwind.css#L4